setup.py

```diff
@@ -3,14 +3,27 @@
 
 # look for environment variable that specifies path to SCIP
 scipoptdir = os.environ.get('SCIPOPTDIR', '').strip('"')
+if not scipoptdir:
+    print('Environment variable SCIPOPTDIR is empty. Please specify path to SCIP.')
+    quit(1)
 
 extra_compile_args = []
 extra_link_args = []
 
 # determine include directory
 if os.path.exists(os.path.join(scipoptdir, 'src')):
-    # SCIP seems to be installed in place
-    includedir = os.path.abspath(os.path.join(scipoptdir, 'src'))
+    # SCIP seems to be installed in place; check whether it was build using make or cmake
+
+    if os.path.exists(os.path.join(scipoptdir, 'src/scip')):
+        # SCIPOPTDIR pointed to the main source directory
+        includedir = os.path.abspath(os.path.join(scipoptdir, 'src'))
+    else:
+        # SCIPOPTDIR probably pointed to a cmake build directory; try one level up
+        if os.path.exists(os.path.join(scipoptdir, '../src')):
+            includedir = os.path.abspath(os.path.join(scipoptdir, '../src'))
+        else:
+            print('SCIPOPT contains a src directory, but we could not find the include files.')
```

Comment:
Maybe better:
```
             print('Location SCIPOPTDIR does not contain a src/scip directory, exiting because we cannot not find the include files.')
```

Reply:
Yes, thanks. Fixed.